src/desktop/apps/partnerships/templates/common_section_mixins.jade

```diff
@@ -44,7 +44,7 @@ mixin support(section)
         p.preamble!= section.preamble
           if path.indexOf('gallery') > 0
             |  You can also read and subscribe to our&nbsp;
-            a(href='/' + sd.GALLERY_INSIGHTS_SLUG) Gallery Insights Newsletter
+            a(href='/gallery-insights') Gallery Insights Newsletter
```

Comment:
I wasn't able to find where this code is rendered on the site -- this could also to to `partners.artsy.net` but I think the naming is misleading if so. Would be glad if anyone is able to confirm that this template is actually rendered on the site at the moment and possibly we can remove.